winpr/libwinpr/sspi/Negotiate/negotiate.c

```diff
@@ -36,17 +36,44 @@
 static const char NEGO_REG_KEY[] =
     "Software\\" WINPR_VENDOR_STRING "\\" WINPR_PRODUCT_STRING "\\SSPI\\Negotiate";
 
+#define ASN_LENGTH_BYTES(len)  \
```

Comment:
That's so unfortunate because we have these computations elsewhere in ber modules (so in FreeRDP). Does it suggest that this should be moved to winPR (any opinion @akallabeth ?)
I'd probably prefer a `static INLINE` function instead of the define with cascading tri-operator.

Reply:
After some thoughts and some reverse inquiries, I think that probably means that we must reintroduce `msasn1` in winPR...